planning/freespace_planner/include/freespace_planner/freespace_planner_node.hpp

```diff
@@ -42,7 +42,12 @@
 #include <nav_msgs/msg/odometry.hpp>
 #include <tier4_planning_msgs/msg/scenario.hpp>
 
+#ifdef USE_TF2_GEOMETRY_MSGS_DEPRECATED_HEADER
```

Comment:
@wep21 It's better to write some descriptions like `Apply the macro added in #720.`